src/compiler/compiler.rs

```diff
@@ -665,7 +665,7 @@ fn detect_compiler<T>(creator: &T,
                       -> SFuture<Option<Box<Compiler<T>>>>
     where T: CommandCreatorSync
 {
-    trace!("detect_compiler");
+    trace!("detect_compiler: {}", executable.display());
```

Comment:
You know, I was just looking at a log file from our CI the other day and realizing that this log message was not very informative. :)